Server/src/openbmp.cpp

```diff
@@ -450,70 +452,93 @@ void runServer(Config &cfg) {
 
                 }
 
-                //TODO: Add code to check for a socket that is open, but not really connected/half open
-            }
+		if(!thr_list.at(i)->baselineTimeout){
```

Comment:
This is causing a SIGABRT, resulting in a core.   The issue is that in the previous if block the thread list entry is removed if the thread isn't running anymore.  This results in the check in thr_list.at(i) being invalid.   The fix is to change this to an ```else if ...```